doc/release/upcoming_changes/25388.new_feature.rst

```diff
@@ -0,0 +1,5 @@
+`numpy.linalg.cholesky` and `numpy.linalg.pinv` new parameters
+--------------------------------------------------------------
+
+``upper`` keyword parameter was added to `numpy.linalg.cholesky` and
+``rtol`` keyword parameter to `numpy.linalg.pinv` for Array API compatibility.
```

Comment:
Weren't both added for array API compat?

I'd also mention that `rtol` doesn't have the array API default.

Reply:
That's correct, both of these were [added to array-api-compat](https://github.com/data-apis/array-api-compat/blob/874c2ff83dcf7d52e4d5a6d1c389cb38d1f8187d/array_api_compat/common/_linalg.py#L86). 

`rtol` doesn't have the default value because default for `pinv` is always taken from `rcond` (for backward compatibility) - I added a note about it.

Reply:
I'm trying to say it reads like only `rtol` was added for compat now. Maybe something like:

```
The ``upper`` and ``rtol`` keyword parameters were added to `numpy.linalg.colesky` and `numpy.lingalg.ping`, respectively, to improve array API compatibility.
```

Reply:
Ah, now I got it - I updated it.